bin/startPipeline.sh

```diff
@@ -221,29 +221,30 @@ function submitPipeline () {
 		if [[ -n "${sampleSheetColumnOffsets['sequencingStartDate']+isset}" ]]
 		then
 			sequencingStartDateIndex=$((${sampleSheetColumnOffsets['sequencingStartDate']} + 1))
-			sequencingStartDate=$(tail -n +2 "${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/${project}.${SAMPLESHEET_EXT}" | awk -v seqstart="${sequencingStartDateIndex}" 'BEGIN {FS=","}{print "$seqstart"}' | head -1)
+			sequencingStartDate=$(tail -n +2 "${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/${project}.${SAMPLESHEET_EXT}" | awk -v seqstart="${sequencingStartDateIndex}" 'BEGIN {FS=","}{print $seqstart}' | head -1)
 		fi
 		if [[ -n "${sampleSheetColumnOffsets['sequencer']+isset}" ]]
 		then
 			sequencerIndex=$((${sampleSheetColumnOffsets['sequencer']} + 1))
-			sequencer=$(tail -n +2 "${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/${project}.${SAMPLESHEET_EXT}" | awk -v sequencer="${sequencerIndex}" 'BEGIN {FS=","}{print "$sequencer"}' | head -1)
+			sequencer=$(tail -n +2 "${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/${project}.${SAMPLESHEET_EXT}" | awk -v sequencer="${sequencerIndex}" 'BEGIN {FS=","}{print $sequencer}' | head -1)
 		fi
 		if [[ -n "${sampleSheetColumnOffsets['run']+isset}" ]]
 		then
 			runIndex=$((${sampleSheetColumnOffsets['run']} + 1))
-			run=$(tail -n +2 "${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/${project}.${SAMPLESHEET_EXT}" | awk -v run="${runIndex}" 'BEGIN {FS=","}{print "$run"}' | head -1)
+			run=$(tail -n +2 "${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/${project}.${SAMPLESHEET_EXT}" | awk -v run="${runIndex}" 'BEGIN {FS=","}{print $run}' | head -1)
 		fi
 		if [[ -n "${sampleSheetColumnOffsets['flowcell']+isset}" ]]
 		then
 			flowcellIndex=$((${sampleSheetColumnOffsets['flowcell']} + 1))
-			flowcell=$(tail -n +2 "${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/${project}.${SAMPLESHEET_EXT}" | awk -v flowcell="${flowcellIndex}" 'BEGIN {FS=","}{print "$flowcell"}' | head -1)
+			flowcell=$(tail -n +2 "${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/${project}.${SAMPLESHEET_EXT}" | awk -v flowcell="${flowcellIndex}" 'BEGIN {FS=","}{print $flowcell}' | head -1)
 		fi
-		_filePrefix="${sequencingStartDate}_${sequencer}_${run}_${flowcell}"
+		
 		if [ "${_sampleType}" == "DNA" ]
 		then
 			capturingKitIndex=$((${sampleSheetColumnOffsets['capturingKit']} + 1))
-			capturingKit=$(tail -n +2 "${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/${project}.${SAMPLESHEET_EXT}" | awk -v capt="${capturingKitIndex}" 'BEGIN {FS=","}{print "$capt"}' | awk 'BEGIN{FS="/"}{print "$2"}' | head -1)
+			capturingKit=$(tail -n +2 "${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/${project}.${SAMPLESHEET_EXT}" | awk -v capt="${capturingKitIndex}" 'BEGIN {FS=","}{print $capt}' | awk 'BEGIN{FS="/"}{print $2}' | head -1)
 		fi
+		filePrefix="${sequencingStartDate}_${sequencer}_${run}_${flowcell}"
```

Comment:
Waarom is _filePrefix veranderd in filePrefix zonder _ aan het begin? Het is voor zover ik kan zien een variabele die alleen in deze functie gebruikt wordt en geen globale, dus zou die volgens conventie eenn lokale scope moeten krijgen en moeten beginnen met een _:
```
local _filePrefix
if ...
	_filePrefix=...
elif ...
	_filePrefix=...
fi
```
Idem voor een groot aantal andere variabelen in deze functie, waaronder: prio, capturingKitIndex, capturingKit, sequencingStartDateIndex, sequencingStartDate, sentrixBarcode_A_Index, sentrixBarcodeA, etc.